tests/fft.cpp

```diff
@@ -29,6 +29,31 @@ struct DFDim : ddc::PeriodicSampling<Kx>
 {
 };
 
+template <typename X, std::size_t Nx>
+static void test_fourier_mesh()
+{
+    double const a = -10;
+    double const b = 10;
+
+    DDom<DDim<X>> const x_mesh(ddc::init_discrete_space<DDim<X>>(DDim<X>::template init<DDim<X>>(
+            ddc::Coordinate<X>(a + (b - a) / Nx / 2),
+            ddc::Coordinate<X>(b - (b - a) / Nx / 2),
+            DVect<DDim<X>>(Nx))));
+    ddc::init_discrete_space<DFDim<ddc::Fourier<X>>>(
+            ddc::init_fourier_space<DFDim<ddc::Fourier<X>>>(ddc::DiscreteDomain<DDim<X>>(x_mesh)));
+
+    double const epsilon = 1e-14;
+    for (int i = 0; i < Nx; i++) {
```

Comment:
Isn't the role of `FourierMesh` to generate the `DiscreteDomain` related to Fourier ?

Reply:
Yes, but I would say we don't need it in 1D. `FourierMesh` is more to generate the appropriate mesh to perform the FFT after ward (taking in account C2C or not, etc.)

Reply:
Yes but then we are creating `DiscreteElement` from raw integers. It is like using `uid()`.

Reply:
[9fac769](https://github.com/CExA-project/ddc/pull/541/commits/9fac769182a53dd73e45950f8d045bf09da7c045) ?